tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -264,14 +255,12 @@ def handler(event: Dict[str, Any], context) -> None:
     """
     Lambda handler. This lambda calls the task function to perform db queries
     and send message to SQS.
-
```

Comment:
Not sure why you are removing these empty lines. They provide whitespace making the description easier to read.

Reply:
added back